src/main/java/utils/Checker.java

```diff
@@ -0,0 +1,33 @@
+package utils;
+
+import models.Ad;
+import models.Email;
+
+/**
+ * This class will be used to validate inputs.
+ * 
+ * @author <a href="https://github.com/Trustacean">Edward</a>
+ */
+
+public class Checker {
+
+    public static boolean passwordLengthValidation(String password) {
+        if (password.length() == 8) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean containsSymbol(String input) {
```

Comment:
`containsSymbol` is a bit unintuitive, since it checks whether it's a letter or digit. How about changing the method name to `isLetterOrDigits`. Return false if contains anything other than letter or digits.

Reply:
I feel like changing the method to isLetterOrDigist makes less sense becase the method is used to check strings (plural)

Reply:
then, `areLettersOrDigits`?